client/js/lounge.js

```diff
@@ -750,6 +763,18 @@ $(function() {
 		case "user":
 			$(".channel.active .users .user[data-name=" + $(this).data("data") + "]").click();
 			break;
+		case "user-ban":
+			socket.emit("input", {
+				target: chat.data("id"),
+				text: "/mode +b " + $(this).data("data")
+			});
+		// eslint-disable-next-line no-fallthrough
```

Comment:
I would prefer we had 3 seperate options (as there is an example of wanting to ban and not kick)

Kick, Ban and Kickban (and I would say "Kickban" rather than "Ban & Kick" because /kickban is a custom command on man servers, so people will know what kickban means).